client.go

```diff
@@ -400,6 +405,7 @@ func (c *Client) Do(req *Request, resp *Response) error {
 			TLSConfig:                     c.TLSConfig,
 			MaxConns:                      c.MaxConnsPerHost,
 			MaxIdleConnDuration:           c.MaxIdleConnDuration,
+			MaxIdemponentCallAttempts: 	   c.MaxIdemponentCallAttempts,
```

Comment:
Can you `go fmt` the code then I can accept it.

Reply:
sorry,my mistake